docs/settings/storagenetwork/manual.md

```diff
@@ -0,0 +1,309 @@
+---
+sidebar_position: 1
+sidebar_label: Manual Storage Network Configuration
+title: ""
+---
+
+# Manual Storage Network Configuration 
+
+This page describes the manual method to configure Longhorn Storage Network in Harvester.
+
+## Prerequisite
+
+1. Make sure that Harvester version is v1.1.0 or above.
+2. Prepare physical interface networks for the storage network, and ensure the VLAN configuration on the external switch side is correct.
+3. Create Network Config in Web UI with the physical interfaces that you expected for the storage network, and remember the `Cluster Network` field for the next step.
+4. Prepare an IPv4 subnet range in the CIDR format. IP Range should be IPv4 CIDR format and 4 times the number of your cluster nodes. This IP range should not overlap with Kubernetes Cluster IPs and Pods IPs.
+
+### Example Configuration
+
+In the following configuration steps, we will use the below configurations for example.
+
+Network Config:
+- ClusterNetwork: test
+- NICs
+  - eth2
+  - eth3
+
+VLANs:
+- Namespace: default
+- Name: storagenetwork-1
+- VLAN ID: 100
+
+IP Range:
+- 192.168.0.0/24
+
+## Configuration
+
+### Step 1: Create VLANs
+
+- Create Cluster Networks/Configs in Web UI.
+    - Choose Networks > Cluster Networks/Configs in the left sidebar.
+- Create VLANs via kubectl command below with desired namespace, name, and VLAN ID, and keep the namespace and name for further configurations.
+- Put Name in name field.
+- Put Namespace in namespace field.
+- Put ClusterNetwork name with `-br` suffix in "bridge" field.
+- Put IP Range into `ipam.range` field.
```

Comment:
```suggestion
- Create Cluster Networks/Configs in Web UI.
    - Choose **Networks > Cluster Networks/Configs** in the left sidebar.
- Create VLANs via kubectl command below with the desired namespace, name, and VLAN ID, and take note of the namespace and name for further configurations.
- Specify the name in the **Name** field.
- Specify the namespace in the **Namespace** field.
- Set **Bridge** to the ClusterNetwork name with the `-br` suffix.
- Set **IP Range** to `ipam.range`.
```

Reply:
Not sure if I understand line 43-46 correctly. Please double-check.

Reply:
It should be ok, thanks for suggestions